src/scenarios/init.ps1

```diff
@@ -1,24 +1,34 @@
 Param(
-    $Channel = 'master' # default is master
+    [string] $InstallDotnetFromChannel,
+    [string] $DotnetDirectory
 )
 
+If ($DotnetDirectory -eq ""){
+    $DotnetDirectory = Join-Path $PSScriptRoot '..\..\tools\dotnet\x64'
+}
+
+# Add scripts and current directory to PYTHONPATH
 $scripts = Join-Path $PSScriptRoot '..\..\scripts' -Resolve
 $env:PYTHONPATH="$scripts;$PSScriptRoot"
-$dotnetScript= Join-Path "$scripts" 'dotnet.py' -Resolve
-$dotnetDirectory = Join-Path $PSScriptRoot '..\..\tools\dotnet\x64'
 
-Write-Host "Downloading dotnet from channel $Channel"
-python $dotnetScript install --channels $Channel -v
+$channel = $InstallDotnetFromChannel
 
-if (!$?){
-    Write-Host "Dotnet installation failed."
-    Exit 1
+If (($channel -ne "")){
+    # Download dotnet from the specified channel
+    Write-Host "Downloading dotnet from channel $channel"
+    $dotnetScript= Join-Path "$scripts" 'dotnet.py' -Resolve
+    python $dotnetScript install --channels $channel
```

Comment:
Can you add back the -v here? I think this is a more informative way to a) let the user know something happened and b) show which version of .NET is being used.